### PR TITLE
[PR] Skeleton structure

### PR DESCRIPTION
**Pull request by philcable**
I had started this months ago, but figured it would be easier to create a new branch rather than continue with the old one :)

### CODE REVIEW

**Comment by philcable** [comment]
@jeremyfelt #reviewmerge?

**Comment by philcable** [comment]
@jeremyfelt #reviewmerge?

File: Gruntfile.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                dest: "js/admin-edit.min.js"
+            },
+            edit_profile: {
+                src: "src/js/admin-profile.js",
+                dest: "js/admin-profile.min.js"
+            }
+        }
+    } );
+
+	grunt.loadNpmTasks( "grunt-contrib-jshint" );
```
Comment (jeremyfelt) [inline Gruntfile.js]: Mixed tabs/spaces! :see_no_evil:

**Comment by philcable** [comment]
@jeremyfelt Good catch! Updated - #reviewmerge?

**Pull request merged**